content/creating-user-centred-content/content-design-subject-experts/content-tips-subject-experts.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: text/textblock
+---
+Having a basic understanding of content design will help you communicate information to your users effectively. It will save you time as there will be fewer changes to the content and your content will most likely meet a user need.
+ 
+### [2]Think about your user
+Don’t make assumptions about users. Use existing research or get a user researcher to investigate your users and what their needs are.
+ 
+Remember, users don't want to be on a government website. Usually they have to be there to complete a task or access information.
+ 
+They shouldn't have to know a department's organisational structure to find what they need. You need to connect information and services in a way that makes sense to users.
+ 
+You can then [test your content](/creating-user-centred-content/testing-content/) to make sure users can do what they need to do.
+ 
+### [2]Benefits of content design
+Content design is about creating well-structured and clear content that meets a user need. Read more about [content design](/creating-user-centred-content/content-design/).
+ 
+Well-designed content will help you:
+- communicate information to your user quickly and easily
+- save time, money and people by helping users access information they need through content, rather than call centres or face-to-face help
+- create content that isn’t rejected by web publishing teams for style, accessibility or online publishing reasons
```

Comment:
I've flagged with the designer these 3 bullets don't sound right with the intro to the list.